gulpfile.js

```diff
@@ -41,23 +41,7 @@ var paths = {
     'bower_components/habitrpg-shared/script/userServices.js',
     'node_modules/habitrpg/public/js/services/groupServices.js',
     'bower_components/habitrpg-shared/script/directives.js',
-    'scripts/services/notificationServices.js',
-    'scripts/services/storeService.js',
-    'scripts/controllers/userAvatarCtrl.js',
-    'scripts/controllers/rootCtrl.js',
-    'scripts/controllers/settingsCtrl.js',
-    'scripts/filters/filters.js',
-    'scripts/controllers/authCtrl.js',
-    'scripts/controllers/tasksCtrl.js',
-    'scripts/controllers/chatCtrl.js',
-    'scripts/controllers/inventoryCtrl.js',
-    'scripts/controllers/petDetailsCtrl.js',
-    'scripts/controllers/mountDetailsCtrl.js',
-    'scripts/controllers/stableCtrl.js',
-    'scripts/controllers/partyCtrl.js',
-    'scripts/controllers/notificationCtrl.js',
-    'scripts/controllers/guildCtrl.js',
-    'scripts/controllers/purchaseCtrl.js'
+    'scripts/**/*.js'
```

Comment:
So strange, that shouldn't have happened. Anyway, https://github.com/HabitRPG/habitrpg-mobile/commit/74a28abd197cd1d8f990129c46f77114b326dc21 . Everything else look ok?